astrocut/asdf_cutouts.py

```diff
@@ -6,6 +6,7 @@
 import asdf
 import astropy
 import gwcs
```

Comment:
I noticed in `get_center_pixel` we are redefining `gwcs`. And also on line `154`. Can you fix?

Reply:
Good catch!  Done.